rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,214 @@
+
+##### Shared Example
+
+Save the `docker-compose.yml` file below to a directory. Notice that no modifiers are added to the volume; stack scope is default behavior.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - data:/data
+volumes:
+  data:
+    driver: local
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+To avoid name collisions with other volumes, unique names are used for the stack scope volumes you define. They include the stack name, original volume name, and a random component. [List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) and identify the `data` volume exists in this form.
+
+> **Note:** Stack-scope volumes could be mounted by other stacks. Stack scope is not a security mechanism. It is solely for managing the lifecycle of the volume.
+
+### V1 Compose
+
+Up until this point, we've discussed volumes in the context of [Docker V2 Compose](https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes-volume_driver).
+
+[Docker V1 Compose](https://docs.docker.com/compose/compose-file/compose-file-v1/#volumes-volume_driver) may also be used. In V1 compose, there is no top-level volume section and volumes can only be defined within the context of a service. Rancher passes V1 volumes directly to Docker; as such, there are no scheduling guarantees and volumes are not automatically deleted. To bridge the gap and ensure containers are scheduled to the host where a V1 volume resides, [scheduling affinities](https://docs.rancher.com/rancher/v1.6/en/cattle/scheduling/#fields-1) may be used.
+
+> **Note:** V1 Compose should be avoided when possible.
+
+#### Pure V1 Compose
+
+Save the `docker-compose.yml` file below to a directory. Notice the absence of the volume section; this doesn't exist in V1. 
+
+```yaml
+etcd:
+  image: rancher/etcd:v2.3.7-11
+  volumes:
+  - etcd:/pdata
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+#### V2 Compose Mashup
+
+Docker compose V2 is a superset of V1; the two schemas can be used interchangeably. Create an environment scope volume first.
+
+```
+$ rancher volume create --driver local etcd_backup
+```
+
+Save the `docker-compose.yml` file below to a directory.
+
+```yaml
+version: '2'
+services:
+  etcd:
+    image: rancher/etcd:v2.3.7-11
+    environment:
+      EMBEDDED_BACKUPS: 'true'
+      BACKUP_PERIOD: 5s
+      BACKUP_RETENTION: 15s
+    volumes:
+    - etcd:/pdata
+    - etcd_backup:/data-backup
+volumes:
+  etcd_backup:
+    driver: local
+    external: true
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+In this example, `etcd_backup` is a V2 environment scope volume and `etcd` is a V1 volume. The absence of a volume definition implicitly makes `etcd` a V1 volume.
```

Comment:
Add:

Finally, note that while you can define an empty volumes section like this:
```
version: '2'
volumes: {}
```
doing so will have no impact. It is equivalent to completely omitting the volumes section. Any volumes referenced by services will still be V1 volumes.